controllers/controllers/workloads/processes/controller.go

```diff
@@ -236,28 +238,44 @@ func (r *Reconciler) createOrPatchAppWorkload(ctx context.Context, cfApp *korifi
 		return err
 	}
 
-	actualAppWorkload := &korifiv1alpha1.AppWorkload{
+	appWorkload := &korifiv1alpha1.AppWorkload{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace: cfProcess.Namespace,
 			Name:      getDesiredAppWorkloadName(cfApp, cfProcess),
+			Namespace: cfProcess.Namespace,
 		},
 	}
 
-	var desiredAppWorkload *korifiv1alpha1.AppWorkload
-	desiredAppWorkload, err = r.generateAppWorkload(actualAppWorkload, cfApp, cfProcess, cfBuild, appPorts, envVars)
-	if err != nil {
-		log.Info("error when initializing AppWorkload", "reason", err)
+	if err = r.modifyAppWorkload(appWorkload, cfApp, cfProcess, cfBuild, appPorts, envVars); err != nil {
```

Comment:
`modifyAppWorkload` function name sounds a bit weird to me. What this function does is that it applies desired state onto the app workload, which is what reconciliation in general is all about. How about `reconcileAppWorkload`? Or maybe `applyToWorkload`?